src/components/editor/Bindings/FieldSelection/FieldActions.tsx

```diff
@@ -0,0 +1,38 @@
+import { Stack } from '@mui/material';
+import { useBinding_recommendFields } from 'stores/Binding/hooks';
+import FieldActionButton from './FieldActionButton';
+import { CompositeProjection } from './types';
+
+interface Props {
+    bindingUUID: string;
+    loading: boolean;
+    projections: CompositeProjection[] | null | undefined;
+}
+
+export default function FieldActions({
+    bindingUUID,
+    loading,
+    projections,
+}: Props) {
+    const recommended = useBinding_recommendFields();
+
+    return (
+        <Stack direction="row" spacing={1} sx={{ alignItems: 'center' }}>
+            <FieldActionButton
+                bindingUUID={bindingUUID}
+                disabled={!recommended[bindingUUID] || loading}
+                labelId="fieldSelection.table.cta.defaultField"
+                projections={projections}
+                selectedValue="default"
+            />
+
+            <FieldActionButton
+                bindingUUID={bindingUUID}
+                disabled={loading}
+                labelId="fieldSelection.table.cta.excludeAllFields"
+                projections={projections}
+                selectedValue="exclude"
+            />
```

Comment:
These buttons should `disable` when `loading` or when form is active to match the other ctas.

Reply:
Or when `projections` are empty... though that might be better suited to be handled within the component itself.